Added note on TDE presence check (#4368)
EnterpriseDB/docs: product_docs/docs/tde/15/enabling_tde.mdx

Use the special value `-` if you don't want to apply any key unwrapping command.

> <!--- Should this be worded differently? 

Reviewer: I think we will be removing this and the following line

Author: Removed... No idea how the line break appeared, but we should resolve questions about content without leaving long lived and untracked comments in the docs.

   data_encryption_key_unwrap_command = 'openssl enc -d -aes-128-cbc -pass pass:ok -in %p'
   ```

## Checking for TDE presence using SQL

The presence of TDE on a server can be determined by querying the `data_encryption_version` column of the `pg_control_init` table. 

> A value of 0 means TDE is not enabled. Any non-zero value reflects the version of TDE in use. Currently, when TDE is enabled, this value will be 1.

Reviewer 2: ```suggestion
A value of 0 means TDE isn't enabled. Any non-zero value reflects the version of TDE in use. Currently, when TDE is enabled, this value is 1.
```

   data_encryption_key_unwrap_command = 'openssl enc -d -aes-128-cbc -pass pass:ok -in %p'
   ```

> ## Checking for TDE in use using SQL

Reviewer: Can we reword this like - Checking for TDE presence using SQL. or something else? Intend is not to use "use" along with using.

Author: Done
